packages/frontend/src/components/svg/SwapIcon.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const SwapIcon = ({ color }) => {
+    const stroke = color || "#fff";
+    return (
+        <svg
+            width="22px"
+            height="22px"
+            viewBox="0 0 21 21"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <g
+                fill="none"
+                fill-rule="evenodd"
+                stroke={stroke}
+                stroke-width="2"
+                stroke-linecap="round"
+                stroke-linejoin="round"
```

Comment:
I think we need to update these to be camel case:

```suggestion
                fillRule="evenodd"
                stroke={stroke}
                strokeWidth="2"
                strokeLinecap="round"
                strokeLinejoin="round"
```